spec/api/v1/authorities_api_spec.rb

```diff
@@ -9,17 +9,15 @@
     subject(:call) { get path }
 
     let(:detail) { 'metadata' }
-    let(:authority_id) { -1 }
```

Comment:
grape does not accepts negative ids in URL anymore, so I've had to update specs to use positive not-existing ids